docs/elements.md

```diff
@@ -1592,6 +1592,7 @@ def generate(data):
 | `params-type`               | string  | `adjacency-matrix` | How to interpret the input data in `params-name-matrix`. By default, only `adjacency-matrix` exists but custom types can be added through extensions.                                                                                                                   |
 | `weights`                   | boolean | `None`             | When using an adjacency matrix, whether or not to show the edge weights. By default will automatically show weights for stochastic matrices (when they are not binary `0`/`1`).                                                                                         |
 | `weights-digits`            | integer | `"2"`              | When using an adjacency matrix, how many digits to show for the weights.                                                                                                                                                                                                |
+| `parse-negative-weights`    | boolean | false              | Whether to recognize negative weights in an adjacency matrix. If set to false, then all weights at most 0 are ignored (not counted as an edge). If set to true, then all weights that are not `None` are recognized.                                                    |
```

Comment:
Will change the name to `negative-weights`.

So the issue is that, the current behavior (equivalent to setting this to false) uses anything nonpositive (0 or negative) to indicate that an edge should not be displayed. However, with this set to true, any negative or 0 value will be displayed as part of the graph. So if this is defaulted to true, then existing uses of this element will display the complete graph with some edges set to weight 0. This issue is actually why I had to use `None` to indicate no edge in the negative edge weight case.

Reading over this again, it may also be worth changing the default behavior to allow `None` to indicate no edge in the non-negative case for consistency.